ArduCopter/AP_Arming.cpp

```diff
@@ -596,6 +596,18 @@ bool AP_Arming_Copter::arm_checks(AP_Arming::Method method)
         return false;
     }
 
+    const NavEKF2 &ekf2 = AP::ahrs_navekf().get_NavEKF2_const();
+    const NavEKF3 &ekf3 = AP::ahrs_navekf().get_NavEKF3_const();
+
+    // Do not arm if EKF lane 1 is not used as primary during arming, when using a certain EKF (EKF2 or EKF3).
+    if (ahrs.get_ekf_type() == 2 && ekf2.currentPrimaryLane() != 0) {
```

Comment:
Could you add parentheses for better readability?

if ((ahrs.get_ekf_type() == 2) && (ekf2.currentPrimaryLane() != 0))

Reply:
`ahrs.get_ekf_type() == 2`

Can '2' be #defined or made a const expr?

Reply:
`ekf2.currentPrimaryLane() != 0`

Can '0' be a #define or const expr?

Reply:
Done!